jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -1,4 +1,5 @@
 def testOptions = [:]
+final QE_REPO_URL = 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
```

Comment:
Is this repo being versioned now?

Reply:
It is

Reply:
:+1: I assume this will use the same versioning as the installer? master-> master v1.3 -> v1.3 etc....

Reply:
Yes. OK I think this is related to your second comment below right? We will basically try to find out a QE repo branch with the same name as the installation branch. Did I get it correctly?

Reply:
Yes exactly. We want the versioning to be the same so that we can easily identify which tests should work with which version of integreatly.